### PR TITLE
update to angular 13

### PR DESCRIPTION
**Pull request by dfa1234**
library build with ng-packagr

### CODE REVIEW

**Comment by dfa1234** [comment]
#55 Hi here what I propose

File: package.json [hunk 1]
```diff
@@ -1,63 +1,50 @@
...
+  ],
+  "peerDependencies": {
+    "@angular/common": "^13.1.2",
+    "@angular/compiler": "~13.1.2",
+    "@angular/core": "^13.1.2",
+    "rxjs": "~7.5.2"
+  },
+  "dependencies": {
+    "exifr": "^4.3.4",
+    "pica": "^9.0.1",
```
Comment (bergben) [inline package.json:34]: does this work? did you test the library? 

Back in the days I had to use a custom wrapper for it, see bergben/ng2-pica

Comment (bergben) [reply package.json:34]: @dfa1234

**Comment by bergben** [comment]
Nice @dfa1234 thanks! 

Did you test the library? Want to release a new version?

**Comment by bergben** [comment]
A new Demo on Plunkr (or now rather Stackblitz) would be nice? See readme

**Comment by dfa1234** [comment]
@bergben You are perfectly right, before all I wanted to prepare a demo on stackblitz. I hope there will be no regression! 

For you question on ng2-pica, the problem is that I could not do `npm i ng2-pica` in this project anymore. I think the problem is that `ng2-pica` need to be upgraded as well with the new angular. To move quicker I simply took your service from ng2-pica and moved it here, made some upgrade on the script (typescript related) and imported pica directly. It's the main change I did in the source code.

I keep you updated when I have a stackbliz working, but I'm also very short on time likely, I expect to work on it next week.

**Pull request merged**